telco-hub/install/mirror-registry/imageset-config.yaml

```diff
@@ -63,13 +60,7 @@ mirror:
     - name: recipe
       channels:
       - name: stable-4.18
-    - name: openshift-gitops-operator
-      channels:
-      - name: gitops-1.15
```

Comment:
We definitely need gitops included but should include both the latest and gitops-1.16 channels.